src/pages/DeveloperHome.jsx

```diff
@@ -0,0 +1,193 @@
+  }, [pathname]);
+  return (
+    <>
+      <Section>
+        <div
+          style={
+            {
+              desktop: {
+                display: "grid",
+                gridTemplateColumns: "repeat(3, 1fr)", // Creates three equal columns
+                columnGap: "40px",
+                rowGap: "20px", // Adds some space between rows
+              },
+              tablet: {
+                display: "flex",
+                flexDirection: "column",
+                rowGap: "3rem",
+              },
+              mobile: {
+                display: "flex",
+                flexDirection: "column",
+                rowGap: "2rem",
+              },
+            }[displayCategory]
+          }
+        >
+          {devTools.map((tool, index) => (
+            <ToolsCard key={index} tool={tool} />
+          ))}
+        </div>
+      </Section>
+    </>
+  );
+};
+
+export default DeveloperHome;
+
+function ToolsCard({ tool }) {
+  const displayCategory = useDisplayCategory();
+  const mobile = displayCategory === "mobile";
+  const tablet = displayCategory === "tablet";
+
+  return (
+    <ToolsStruct
+      tablet={tablet}
+      mobile={mobile}
+      top={
+        <div
+          style={{
+            width: mobile ? "150px" : tablet ? "350px" : "100%",
+            height: mobile ? 150 : 300,
+          }}
+        >
+          {" "}
+          {/* Set width to 100% */}
+          <img
+            style={{
+              width: "100%",
+              height: "100%",
+              objectFit: "cover",
+            }}
+            src={tool.image}
+            alt=""
+          />
+        </div>
+      }
+      bottomRight={
+        <div
+          style={{
+            marginRight: !mobile && 10,
+            marginBottom: !mobile && 10,
+            height: mobile && "100%",
+          }}
+        >
+          <LinkButton
+            text="Open"
+            link={`${tool.path}`}
+            width="100%"
+            height={"100%"}
+          />
+        </div>
+      }
+      style={{
+        backgroundColor: style.colors.LIGHT_GRAY,
+        // maxWidth: "1000px", // Set a max width for ToolBox
+        height: "100%",
+        position: "relative",
+      }}
+    >
+      <div
+        style={{
+          padding: "0",
+          minHeight: 100,
+          width: "100%", // Change to 100% for grid layout
+        }}
+      >
+        <h4
+          style={{
+            paddingTop: "1rem",
+            paddingLeft: "1rem",
+            paddingRight: "1rem",
+            width: `${mobile ? "150px" : "100%"}`,
+          }}
+        >
+          {tool.title}
+        </h4>
+        {displayCategory !== "mobile" && (
+          <p
+            style={{
+              paddingLeft: "1rem",
+              paddingTop: "10px",
+              paddingRight: "1rem",
+            }}
+          >
+            {tool.desc}
+          </p>
+        )}
+      </div>
+    </ToolsStruct>
+  );
+}
+
+function ToolsStruct({
```

Comment:
Just personal preference: structs do exist in JS (albeit more as a concept) could you change this name?